Makefile

```diff
@@ -11,7 +11,7 @@
 #   Stable releases:   "1.0.0"
 #   Pre-releases:      "1.0.0-alpha.1", "1.0.0-beta.2", "1.0.0-rc.3"
 #   Master/dev branch: "1.0.0-dev"
-VERSION=7.0.0-beta.1
```

Comment:
Are all these version changes necessary here?

Reply:
They're for cutting a dev build but aren't needed in the PR itself, will remove.